src/dataprotection/HISTORY.rst

```diff
@@ -2,6 +2,15 @@
 
 Release History
 ===============
+1.4.0
```

Comment:
If you want to release a new version after this PR gets merged, don't forget to upgrade the version in setup.py

Reply:
updated setup.py